test/test_utils/__init__.py

```diff
@@ -862,6 +863,9 @@ def parse_canary_images(framework, region):
                 versions_counter[version]["tr"] = True
             elif "inf" in tag_str:
                 versions_counter[version]["inf"] = True
+            
+            if match.group(3):
```

Comment:
you don't need this condition, right - it's not an optional parameter in the regex

Reply:
Done, thanks